conjur/cli.py

```diff
@@ -272,7 +361,7 @@ def _parse_args(parser):
             sys.exit(0)
 
         # Check whether we are running a command with required additional arguments/options
-        if args.resource not in ['list', 'whoami', 'init']:
+        if args.resource not in ['list', 'whoami', 'init', 'login', 'logout']:
```

Comment:
`init` and `logout` does not get here, so no need to put them.

Reply:
They do need to go here because they can accept other arguments. We don't run `conjur login` and execute the command (like we do with variable or policy). It is an interactive command and we need more input.
login / logout are classified as a 'resource' so need to be here. 

If I remove them, I get the help screen

Reply:
Couldn't fully understand, but I trust you.